lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -195,7 +196,7 @@ private void sendIdentityIssuedAuditEvent(
                         ipvSessionItem.getVot(),
                         ipvSessionItem.isCiFail(),
                         contraIndicators.hasMitigations(),
-                        userIdentity.getExitCode());
+                        configService.enabled(EXIT_CODES) ? userIdentity.getExitCode() : null);
```

Comment:
I'm not sure this is totally necessary - the exit code should already be `null` if the flag is not enabled. I suppose it adds an extra layer of certainty.

Reply:
Ticket says - Returning Failure Codes in the IPV_IDENTITY_ISSUED audit event is Controlled with a Feature Toggle

If not needed then will close this PR.

Reply:
As part of 3847 feature flag is removed. Will close this and then update the ticket accordingly.